subprojects/build-init/src/main/groovy/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -63,6 +64,7 @@
     private PathBase distributionBase = PathBase.GRADLE_USER_HOME;
     private String distributionUrl;
     private GradleVersion gradleVersion;
+    private String distType;
```

Comment:
So while working on this I realized that it seems I cannot access `Wrapper.DistributionType` from `DistributionLocator` as they are in different subprojects ...